app/jobs/order_cycle_opened_job.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Trigger jobs for any order cycles that recently opened
+class OrderCycleOpenedJob < ApplicationJob
+  def perform
+    ActiveRecord::Base.transaction do
+      recently_opened_order_cycles.find_each do |order_cycle|
+        SubscriptionPlacementJob.perform_later(order_cycle.id)
+        OrderCycleWebhookJob.perform_later(order_cycle.id, 'order_cycle.opened')
```

Comment:
This type of event 'order_cycle.opened' should be extracted to constant somewhere.

Reply:
I'm not sure where/how exactly yet, but will try to address this in my next update next week.